dces-drc-integration/src/main/resources/schemas/contributionsLine.xsd

```diff
@@ -517,7 +516,7 @@
                                     <xsd:element type="xsd:string" name="status"/>
                                     <xsd:element type="xsd:date" name="bsStartDate"/>
                                     <xsd:element type="xsd:date" name="bsEndDate"/>
-                                    <xsd:element type="xsd:integer" name="debtAmt"/>
+                                    <xsd:element type="xsd:long" name="debtAmt"/>
```

Comment:
DB stores no decimal places, but can hold 10 digits.

Reply:
`xsd:integer` is unlimited precision (it's only `xsd:int` that's limited to 32 bits).